packages/apps/dev-wallet/src/modules/account/account.repository.ts

```diff
@@ -87,15 +98,23 @@ const createAccountRepository = ({
     addAccount: async (account: IAccount): Promise<void> => {
       return add('account', deleteKey(account, 'keyset' as const));
     },
+
     updateAccount: async (account: IAccount): Promise<void> => {
       return update('account', deleteKey(account, 'keyset'));
     },
+    deleteAccount: async (uuid: string): Promise<void> => {
+      return remove('account', uuid);
+    },
     getAccount: async (id: string) => {
       const account: IAccount = await getOne('account', id);
-      return appendKeyset(account);
+      return appendKeyset(account as IAccount);
```

Comment:
Why is this `as IAccount` necessary? The `const account` is already typed as `IAccount`. 
I saw the `as IAccount` in more places.

Reply:
nice catch! I was doing something that I removed eventually but forgot to clean up this line